force_wfmanager/ui/setup/new_entity_creator.py

```diff
@@ -1,3 +1,5 @@
+from envisage.plugin import Plugin
+
```

Comment:
assuming that my interpretation of the import order rules is correct (thanks for the trust!), this blank line shouldn't be there (both ETS packages)